lms/djangoapps/grades/transformer.py

```diff
@@ -49,6 +50,12 @@ def collect(cls, block_structure):
         """
         block_structure.request_xblock_fields(*cls.FIELDS_TO_COLLECT)
         cls._collect_max_scores(block_structure)
+        collect_unioned_set_field(
+            block_structure=block_structure,
+            transformer=cls,
+            merged_field_name='containing_subsections',
```

Comment:
How hard will it be to generalize this for people (was it courses, or openedx installations?) who want to aggregate scores by vertical instead of subsection?